### DISCUSSION TITLE
BO2 TOWN [NEW UPDATE]

### DISCUSSION THREAD

**Discussion by glitcheking** [nzp-team/nzportable#470] [Map Releases]
**NEW UPDATE 27/08/2024**
This is a remake of black ops 2 town
The map is fully working on pc and psp! i optimized it a bit more changed the textures
also made it more diffecult  
can you find a way to open the vault!!

The map FEATURES.

PAP.
13 WALLBUY GUNS.
DOGS.
CUSTOM SKYBOX
FULL EAST EGG 
4 TEDDYS 
TRAPS
CHRISMAS LIGHTS 
ALL 7 PERKS
4 player coop 

Wat will Come in FUTURE UPDATES.
HURTING LAVA 
Custom sounds 
Full underground part of the vault
More props 
And a lot more!

### [DOWNLOAD HERE](https://archive.org/download/town-update/town%20update.zip)
.
.
.

![Nazi Zombies_ Portable 26_08_2024 12_12_16](https://github.com/user-attachments/assets/1417f4a5-e3c1-4213-b10f-cd5b961e7ccf)

![Nazi Zombies_ Portable 26_08_2024 12_11_52](https://github.com/user-attachments/assets/8dece13e-7349-4d51-9b88-4f7ac237bec7)

![Nazi Zombies_ Portable 26_08_2024 12_09_56](https://github.com/user-attachments/assets/b0738811-bc44-40fc-a1b9-cf6e596df76b)

![Nazi Zombies_ Portable 26_08_2024 12_10_44](https://github.com/user-attachments/assets/66cfada9-c074-4e1a-9962-37c3c7ff22f8)

![Nazi Zombies_ Portable 26_08_2024 12_11_11](https://github.com/user-attachments/assets/93cbed7e-6d7c-474b-b442-daa39229c302)

![Nazi Zombies_ Portable 26_08_2024 12_11_33](https://github.com/user-attachments/assets/5fb5aadd-aeed-45be-a3a4-9be83f5087ba)

![Nazi Zombies_ Portable 26_08_2024 12_10_09](https://github.com/user-attachments/assets/2d7bb36a-f288-4cc4-a595-8ab91cd9d912)

**Comment by MotoLegacy** [comment]
Hi there. In order for this to meet release standards, edit the post to make the download button bigger, and clearer. Otherwise LGTM.

**Reply by Bonniegamer202** [reply]
Hello! I play on vita and this sadly crashes every time I try to play it?

**Reply by glitcheking** [reply]
hi there i will change it soon i will also try to make a psp version with cheaper  textures

**Reply by MotoLegacy** [reply]
I thought you said nzp was dead? :)

Just fix your non integer vertices and it will boot. textures aren't the issue.

**Reply by glitcheking** [reply]
yea want to give it another try so i wil try to fix it

**Reply by w1lm34r** [reply]
Hola yo sé que esto es de hace un año pero por si alguna razón están hay será que me pueden decir si este mapa está optimizado para 3ds

**Comment by Phynix0027** [comment]
Hello, i tried to play it on psp, but is imposible. There is no way to make it works?

**Comment by LorenzoMorelli** [comment]
Same here from psvita, it crashes everytime I try to start it :(

**Comment by EVDVMusic** [comment]
Easter Egg doesn't work.

Teddy Locations (That I know):
Teddy on Bar Roof
Teddy in Bank Window
Teddy in Barber Shop

Maybe near cars?
Maybe near Stamin-up?

**Reply by EVDVMusic** [reply]
5/5 no vault

**Reply by EVDVMusic** [reply]
If you can't find all of them toss some grenades

Throw a grenade:
In the barber shop window
In the bank window (inside the bank)
Near Double Tap (Hits the Teddy on the other side)
and on the roof (easy shot ngl)

I have no idea where 4/5 is

**Reply by EVDVMusic** [reply]
SOLVED THE PUZZLE

The teddies are laid in spots with slightly out of place structure

Bar Roof (First Floor) (1/5)
Bank Window Slab Thingy? (2/5)
Sticking-Out Corner in Jug House Window (3/5)
Sticking-Out-Corner near Stamin-Up (4/5)
Sticking-Out-Corner near PhD (5/5)

Bank Vault still doesn't open :c

**Reply by EVDVMusic** [reply]
Rewards that are inaccessible that are in the vault:

Paped .357 Magnum Wallbuy (1000)
Raygun Wallbuy (3000)
FG42 Wallbuy (1000)
MG42 Wallbuy (2000)
Bowie Knife Wallbuy (0)

**Reply by EVDVMusic** [reply]
SORRY DIDN'T SEE YOUR REPLY!!!

1. "Cannot see anything sticking out." They don't show up, just throw grenades at the weird parts. They tend to be in extended parts of the wall.
2. "How'd you find the inaccessible rewards...?" Noclip.

**Reply by conbobcob** [reply]
how do you play this i downloaded but idk how to play

**Reply by glitcheking** [reply]
i wil fix it soon!

**Comment by theidioticdev** [comment]
For everyone who is wondering how to add a custom map to the game:
First: unzip the .zip file that has the map.
Second: after you unzip the file, open the folder and you will find some files like (map, gfx...) copy the files inside these folders and paste each copy to its corresponding folder in the game files, for example:
the files inside (map) folder should be copied to the same folder in the game files.
I hope that helped 👍

**Comment by glitcheking** [comment]
TOWN IS UPDATED!

**Comment by glitcheking** [comment]
i just fixed the zombies getting stuk at quickrevive so new download

**Comment by Renatochon** [comment]
Does anyone know why it won't let me put it on On android psp it doesn't let me, I put it in the nzp folder then in maps and it's the dsp file

**Reply by glitcheking** [reply]
Yes the .bsp

**Reply by glitcheking** [reply]
Oo wait one second you are trying to play der riese witch is currentley not possible only on pc 
I wil update it in 7 days 
But if you wand you can download my other map bo2 town that one is updated and shut work on psp emulator

**Reply by Renatochon** [reply]
And I just put the bsp file in maps?Or I put more

**Reply by Renatochon** [reply]
I'm trying to use the bsp file and it tells me that it's not supported. I'm doing it in zarchive.

**Reply by glitcheking** [reply]
Yes bsp in maps

**Reply by glitcheking** [reply]
> I'm trying to use the bsp file and it tells me that it's not supported. I'm doing it in zarchive.

I think i use 7 zip because its a zip file

**Reply by Renatochon** [reply]
Oo okay thanks

**Reply by Renatochon** [reply]
It's an app, right?

**Reply by glitcheking** [reply]
Yes on android

**Reply by Peter0x44** [reply]
If you want support, please join the Discord server instead, that's not what this thread is for. You shouldn't need any external program that isn't part of windows, but using 7-zip is still a good idea.
EDIT: NVM I didn't read the context. It would still be easier to help you in the first instance.

**Reply by glitcheking** [reply]
Tru on the nzp discord you wil be shown how to do it

**Reply by Renatochon** [reply]
Oo Thank you so much

**Reply by Renatochon** [reply]
Where do I find the discord?

**Reply by kmorris1077** [reply]
https://discord.nzp.gay/

**Comment by FrostyXPP** [comment]
does this map work on browser?

**Reply by glitcheking** [reply]
Hi no in the browser version you cant download custom maps

**Reply by kmorris1077** [reply]
Yes you can.  Steps:

1. Download the zip file from the **Download Here** link above.  
2. Rename the extension to .pk3.
3. In your browser load https://nzp.gay
4. Drag the "town update.pk3"  file it into your browser window.
5. You will be prompted to install it. 
6. Reload the browser tab.
7. It should now appear under Custom Maps in game.  
8. When the game ends I typically just use the console command(~) "restart" to start the next game.

Disclaimer: I'm on a Mac and use Firefox.

EDIT: Corrected the instructions based on MotoLegacy's feedback.

**Reply by MotoLegacy** [reply]
These instructions are incorrect as there is more to a map than the bsp file.

**Reply by kmorris1077** [reply]
Well it results in a very fun playable game.

**Reply by MotoLegacy** [reply]
Without zombie pathfinding or the mystery box weapon allow list and NO custom sounds or models the map utilizes. That is not playable.

**Reply by kmorris1077** [reply]
Ah yes, the Zombies do often get confused about where they are going :).  Do you know if there are any plans to make custom maps like this loadable in the browser edition? Is the current .bsp drag and drop approach just a half baked feature then?

**Reply by MotoLegacy** [reply]
If the map meets the install standard, rename the .zip to .pk3 and drag it into the window.

**Reply by kmorris1077** [reply]
That worked!  Zombies path find now and the skybox image is showing. Thanks! Fixed the instructions.

**Reply by telaphone99** [reply]
> Yes you can. Steps:
> 
>     1. Download the zip file from the **Download Here** link above.
> 
>     2. Rename the extension to .pk3.
> 
>     3. In your browser load https://nzp.gay
> 
>     4. Drag the "town update.pk3"  file it into your browser window.
> 
>     5. You will be prompted to install it.
> 
>     6. Reload the browser tab.
> 
>     7. It should now appear under Custom Maps in game.
> 
>     8. When the game ends I typically just use the console command(~) "restart" to start the next game.
> 
> 
> Disclaimer: I'm on a Mac and use Firefox.
> 
> EDIT: Corrected the instructions based on MotoLegacy's feedback.

Not working for me, its telling me town update.pk3 doesnt exist

**Reply by kmorris1077** [reply]
I just tested it, and it did work for me.  Maybe try renaming the file to remove the space? I.e., town_update.pk3

**Comment by Mr-Roberts01** [comment]
Is the Easteregg working yet?

**Reply by glitcheking** [reply]
No not yet

**Comment by ElDani123** [comment]
we are about to play with some friends, but the downloader is down, can you please upload it trough mediafire?, i search for it in discord o google but no link

**Reply by kmorris1077** [reply]
[town update.zip](https://github.com/user-attachments/files/17387321/town.update.zip)

**Reply by ElDani123** [reply]
thanks

**Comment by Nye12g** [comment]
please can anybody help me download the game on macbookair or tell me the steps to open the map in browser? i would die to play this map

**Reply by Paindeluxe** [reply]
@Nye12g sorry for the late reply but I'll show you how to play it with the Web Version of NZP.

1. You [download](https://archive.org/download/town-update/town%20update.zip) the Map
2. You change the File Extension from .zip to .pk3
3. You open https://nzp.gay/
4. Drag and Drop the "town update.pk3" onto the https://nzp.gay/ Tab
5. Open the Console and type `map town`

Video Guide:

https://github.com/user-attachments/assets/6ee8af0f-5253-4ca7-9187-dd9dd36cbb3e



Also don't mind the fact that I am not recording that Video on a Mac, I am just too lazy to do that. It will work regardless.

**Comment by NOVO-17** [comment]
I wanna get the perks, But it says you have to get the power back up
How do i get the power back?? can someone help me.

**Comment by M4x1W4lk3r** [comment]
is the sky bug happening only on the ps vita?
the sky textures are messed up with only a few random pixels, and there ain't any fog

**Comment by Dantooswag** [comment]
It says sys_loadtexture:too big on 3ds is there a way to fix it?

**Comment by tayyabkhan21** [comment]
hi. wonderful map. I wanted to ask when are you going to update it? and also it would be nice if u make the map larger cos the lava pool and streams are to thin if u know what i mean, like in bo2

**Comment by Kingston4508** [comment]
Anyone open the vault yet?  I keep missing one bear.  Could use some help.

**Reply by EVDVMusic** [reply]
Bar Roof (First Floor) (1/5)
Bank Window Slab Thingy? (2/5)
Sticking-Out Corner in Jug House Window (3/5)
Sticking-Out-Corner near Stamin-Up (4/5)
Sticking-Out-Corner near PhD (5/5)

The problem is that the EE doesn't work.

**Reply by EVDVMusic** [reply]
Use explosive weapons like grenades, Raygun or Panzer to hit the ones hidden in the extended geometry.

**Comment by nonMat** [comment]
So good! 
Gave me nostalgia for when I used to play BO2 as a kid! (It terrified me).
Quite sad that the Easter Egg doesn't work ):

**Comment by Marianoa213** [comment]
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/5a32a126-c588-441d-b936-02882d96aaec" />
got to round 38 with some friends, def took forever from round 20 to 38, but it was hella fun. 9/10 (still missing the lava and the working easter egg, but other than that, it's awesome)

**Comment by Galladite27** [comment]
Got round 26 solo, would have been able to get further but sadly when I ran into the PPSH room my game crashed with some "out of bounds" error. Otherwise great map!

**Comment by FleeingFawn** [comment]
hello! played on PPSSPP for android on a samsung phone. great map! however, i do have some performance issues. mostly lags when looking into the center area at a lil bit of a distance. also i'm unsure if the lava is supposed to be harmless but it doesn't hurt me any.

otherwise i love how you made this map!! very well done

**Comment by sspider27** [comment]
https://github.com/user-attachments/assets/9c03ef48-cd72-4134-9467-5753126484e7

Pathfinding issue

**Comment by sspider27** [comment]
https://github.com/user-attachments/assets/687f38fb-420b-4cf2-89a2-31703f045c5a

Visual Glitches

**Comment by wardog3000** [comment]
Where do I present? This game I downed on the same round getting stuck like a sandwich 
<img width="1838" height="856" alt="17C4A083-8E2C-41BB-B914-4EDC29B47ABA" src="https://github.com/user-attachments/assets/f273027d-bdd2-4df2-bc85-f2ecf169b645" />

**Reply by FADEDAGAIN** [reply]
round 129 with only 11870 points???????

**Comment by bingler38** [comment]
Super laggy on 3ds, not really playable on my og 3ds
